__tests__/features/fn_move_spec.js

```diff
@@ -16,120 +24,127 @@ describe('#move', function () {
     expect(raty.isMove).toEqual(false);
   });
 
-  describe('with interger score', function () {
-    it('moves to the right point', function (done) {
+    describe('with interger score', () => {
+      
+        it('moves to the right point', done => {
       // given
       var raty = new Raty(document.querySelector('#el'), {
         precision: true,
         target: '#target',
         targetType: 'number',
       }).init();
 
-      var that = this;
-
       setTimeout(function () {
         // when
         raty.move(1);
 
         // then
-        expect(that.target.text()).toEqual('1.0');
+        expect(testContext.target.text()).toEqual('1.0');
 
         done();
       }, 100);
+            
+      jest.advanceTimersByTime(100)
```

Comment:
Do we need it? The `setTimeout` was just because sometimes the target was not filled yet.